localization.lua

```diff
@@ -78,6 +80,7 @@ elseif locale == "frFR" then -- translated by Adirelle
 		["Immune"]  = "Immunisé",
 		["PvE"]     = "PvE",
 		["player"] = "Joueur",
+		["pet"]    = "Animal",
```

Comment:
Is that from google translate? I'll change it later

Reply:
KEKW